pkg/kubectl/cmd/label.go

```diff
@@ -57,6 +57,9 @@ $ kubectl label pods foo bar-`
 )
 
 func NewCmdLabel(f *cmdutil.Factory, out io.Writer) *cobra.Command {
+	p := kubectl.NewHumanReadablePrinter(false, false, false, false, []string{})
```

Comment:
It's still something that is incorrect according to our coding guidelines
and patterns in use.   I don't expect this change to fix the entire
codebase, but I do expect it not to make it worse.

On Aug 23, 2015, at 3:30 PM, Eric Paris <notifications@github.com> wrote:

In pkg/kubectl/cmd/label.go
<https://github.com/kubernetes/kubernetes/pull/13047#discussion_r37709620>:

> @@ -57,6 +57,9 @@ $ kubectl label pods foo bar-`
>  )
>
>  func NewCmdLabel(f *cmdutil.Factory, out io.Writer) *cobra.Command {
> +	p := kubectl.NewHumanReadablePrinter(false, false, false, false, []string{})

Agreed, it should be added. But this patch is correct for kube. If others
want to build on it I certainly think they should. This was pointed out to
OpenShift people back when support for get was added...

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/13047/files#r37709620>.